aesthetic_ants/penguin/tile_layer.py

```diff
@@ -121,3 +172,35 @@ def collide_tiles(self, other, _):
                 continue
 
             other.collide_tile(self.tiles[y][x])
+
+    def decay_tiles(self, dt: float, enemies: List[Enemy]):
+        self.decay += DECAY_RATE * dt
+        self.decay += dt * ENEMY_DECAY_INFLUENCE * len(enemies)
```

Comment:
Actually, these don't change in any other method, do they?

You could just make `decay` local.